{{cookiecutter.project_slug}}/clients/vue3/public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
```

Comment:
**Info:** Needed a language defined here for the `lang` attribute for WCAG compliance.

Screen shot of issue from Axe Dev tools in Chrome attached.
![Screen Shot 2023-01-04 at 5 30 42 PM](https://user-images.githubusercontent.com/79381287/210662544-9c54724d-ea6b-4f28-b5d1-d4488f791ad7.png)